packages/containerd-1.7/1003-Allow-sections-of-Plugins-to-be-merged-and-not-overw.patch

```diff
@@ -0,0 +1,31 @@
+From a21e379b69916558c1771b086a4a17b0a58f7cc1 Mon Sep 17 00:00:00 2001
```

Comment:
Can you reference the source commit in containerd? If this was a cherry-pick, you can use `-x`.

Reply:
This is the source commit in containerd: https://github.com/containerd/containerd/commit/a21e379b69916558c1771b086a4a17b0a58f7cc1 I've reworked the commit to apply to 1.7